packages/firestore/src/platform/browser/webchannel_connection.ts

```diff
@@ -134,22 +140,21 @@ export class WebChannelConnection extends RestConnection {
               break;
             default:
               fail(
-                'RPC "' +
-                  rpcName +
-                  '" failed with unanticipated ' +
-                  'webchannel error ' +
+                `RPC '${rpcName}' ${streamId} ` +
+                  'failed with unanticipated webchannel error: ' +
                   xhr.getLastErrorCode() +
                   ': ' +
                   xhr.getLastError() +
                   ', giving up.'
               );
           }
         } finally {
-          logDebug(LOG_TAG, 'RPC "' + rpcName + '" completed.');
+          logDebug(LOG_TAG, `RPC '${rpcName}' ${streamId} completed.`);
         }
       });
 
       const requestString = JSON.stringify(body);
+      logDebug(LOG_TAG, `RPC '${rpcName}' ${streamId} sending request:`, body);
```

Comment:
Are we logging the connection when it is a success as well? How frequent we are expected to make connection requests? Would this populate the log entry too much?

Reply:
Yes. Lines 83-87 log the success of the request. The other `logDebug(...)` lines above log the various failures. So each request would gain one additional log entry due to this particular line being added. IMO that is acceptable and won't spam the logs. What do you think?